javascript/frameworks/ui5/lib/advanced_security/javascript/frameworks/ui5/BindingStringParser.qll

```diff
@@ -431,10 +490,10 @@ module BindingStringParser<BindingStringReaderSig BindingStringReader> {
   private class DotToken extends Token, MkDot { }
 
   private Token getNextSkippingWhitespace(Token t) {
-    result = t.getNext() and
+    result = t.getNextAfterEnd() and
```

Comment:
I'm not very sure if `getNextAfterEnd` should be used in place of `getNext`. `t.getNextAfterEnd` finds a token which may come after a whitespace token that this token may contain, so this may give `bar` when token `t`'s value is `"foo bar"`, whereas the expected behavior of `getNextSkippingWhitespace` is to get the token that comes after the entire `"foo bar"`.